code/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2020.2.2"
+    id "edu.wpi.first.GradleRIO" version "2020.3.2"
```

Comment:
There weren't many changes from this version. If we do move over to this version before comp, we would have to make sure that the new version of wpilib is downloaded/cached on the driver stations.

Reply:
I upgraded as I 'thought' it fixed an issue with the HAL Simulator. However, it didn't seem to make a difference. I'd verify the changes are not helpful to our code:
https://github.com/wpilibsuite/allwpilib/releases

Reply:
Changed it back for now.